timely_beliefs/tests/test_belief_query.py

```diff
@@ -187,7 +187,7 @@ def test_query_belief_by_belief_time(
         len(
```

Comment:
Isn't `len(bdf.index) == 0` equivalent to `bdf.empty`?

Reply:
Also, I think the test could be simplified using a for loop to test for different `belief_time` or the use of `pytest.parametrize`.